### DISCUSSION TITLE
两个快捷键需求

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1613] [功能建议 / Ideas]
1. 参考 YouTube 的 <kbd>Home</kbd>/<kbd>End</kbd> 跳至视频的开头或结尾
2. 一个奇奇怪怪的需求：显示/隐藏自定义顶栏快捷键

**Comment by the1812** [comment]
> 参考 YouTube 的 Home/End 跳至视频的开头或结尾

可以加个跳结尾的, 跳开头的已经有了

> 一个奇奇怪怪的需求：显示/隐藏自定义顶栏快捷键

好家伙, 我要不把快捷键的行为也开放自定义得了

**Reply by Geeyun-JY3** [reply]
> 可以加个跳结尾的, 跳开头的已经有了

配套着加吧，YouTube 也是这样的，单独一个 <kbd>End</kbd> 怪怪的。~~或者学 YouTube 把键盘主键区中的数字 1 到 9 相应跳至视频进度的 10% 到 90% 这套快捷键都做了 0 就不孤单了。~~

> 好家伙, 我要不把快捷键的行为也开放自定义得了

如果能实现我不介意🌚。就是会有多少种行为和维护起来工作量会不会很大🙈。

**Reply by Geeyun-JY3** [reply]
> ~~或者学 YouTube 把键盘主键区中的数字 1 到 9 相应跳至视频进度的 10% 到 90% 这套快捷键都做了 0 就不孤单了。~~

来了噢：#5458
